shims/spark311db/src/main/scala/com/nvidia/spark/rapids/shims/spark311db/Spark311dbShims.scala

```diff
@@ -99,6 +104,17 @@ class Spark311dbShims extends Spark311Shims {
             )
           }
         }).disabledByDefault("it only supports row based frame for now"),
+      GpuOverrides.exec[RunningWindowFunctionExec](
```

Comment:
does it make sense to find a way to combine both shim impls? If not appropriate for this PR, perhaps a follow up issue.

Reply:
much of this I explicitly left as copied when I created it.
1) its safer to build against the specific Databricks version
2) spark 3.2.0 changes will probably end up doing this anyway
3) hopefully the spark301dbshim will go away soon

it is a bit of maintenance overhead but doesn't change much and for this it seems safer just to have 2 copies for a little while until we get to the changes needed to support spark 3.2.0